docs/source/developer/initial_setup.rst

```diff
@@ -0,0 +1,81 @@
+**************************************
+Setting Up CMakePPLang for Development
+**************************************
+
+This page contains information to help developers new to CMakePPLang set up
+the project to begin development.
+
+Downloading the Source Code
+===========================
+
+During development, the latest version of CMakePPLang should be used.
+CMakePPLang is `hosted on GitHub <CMakePPLang_src_>`_ and can be cloned to
+your local system using Git_:
+
+.. code-block:: bash
+
+   git clone https://github.com/CMakePP/CMakePPLang.git
+
+Building CMakePPLang
+====================
+
+CMakePPLang is built on top of and extends the `CMake language <CMake_>`_, so
+"building" CMakePPLang is performed during CMake's configuration step. The
+build step usually required by projects using CMake is not required to
+build CMakePPLang itself, so it will not be included here.
+
+Once the CMakePPLang source code has been downloaded, navigate into the new
+``CMakePPLang`` directory and run the following command to build it:
+
+For \*nix-based systems (including Mac OSX):
+
+.. code-block:: bash
+
+   cmake -H. -Bbuild -DBUILD_TESTING=ON
+
+For Windows systems:
+
+.. code-block:: bash
+
+   cmake -S. -Bbuild -DBUILD_TESTING=ON
+
+In these commands, ``-H`` and ``-S`` specify the top-level, root directory for
+the project, called the "source directory" in CMake (not to be confused with a
+"source code directory", commonly also called the "source directory"). In this
+case, we are assuming that the current directory, ``.``, is the root directory
+of the ``CMakePPLang`` repository. The directory where build artifacts will
+appear is specified by ``-B``, meaning we are directing build artefacts to
+``CMakePPLang/build``. Finally, ``-DBUILD_TESTING=ON`` enables unit testing on
+CMakePPLang, which will be necessary for development. ``BUILD_TESTING``
+defaults to ``OFF``, so this argument can be excluded if testing is not needed.
+
+Running CMakePPLang Tests
+=========================
+
+Unit tests from the build will be included in ``CMakePPLang/build`` with the
```

Comment:
Strictly speaking the unit tests live in the `tests/` dir. What's in the build directory are the CTest scripts for running them.